src/Command/DeleteClientCommand.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace App\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\Console\Input\InputArgument;
+
+class DeleteClientCommand extends BaseCommand
+{
+    protected function configure(): void
+    {
+        parent::configure();
+        $this
+            ->setName('client:delete')
+            ->setDescription('Delete a client')
+            ->addArgument('id', InputArgument::REQUIRED, "Client's id")
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $httpClient = HttpClient::create();
+        $url = $input->getOption('apiUrl');
+        $username = $input->getArgument('username');
+        $password = $input->getArgument('password');
+        $id = $this->parseInt($input->getArgument('id'));
+        $output->writeln("Delete client ".$id);
+        $response = $httpClient->request('DELETE', $url.'/api/clients/'.$id, ['auth_basic' => [$username, $password],]);
+        if (204 == $response->getStatusCode()){
+            $output->writeln("Client ".$id." successfully deleted");
+        } else {
+            $output->writeln($response->getContent());
+        }
+    }
```

Comment:
Command should return 0 if successful, a non-zero value if error. This must be done for all commands. You can check return value of a command typing "echo $?" after the command in a shell.